recipe/meta.yaml

```diff
@@ -25,20 +25,21 @@ requirements:
     - numpy   1.20  # [not (osx and arm64) and py<310]
     - numpy   1.21  # [(osx and arm64) and py<310]
     - numpy   1.21  # [py==310]
+    - numpy   1.23  # [py==311]
```

Comment:
I think this is correct, but a confirmation would be appreciated! Upstrea: https://github.com/alkaline-ml/pmdarima/compare/v1.8.5..v2.0.2#diff-da41c061ccf1a2f3a9e0da75e16c56464409eacafbb8e085fcadd3df5dfc02d7R1-R4.

They also kind of changed their build requirements for all python versions for both numpy and scipy... Should we follow what they did?

Reply:
Usually numpy/scipy reqs are more complicated upstream because they take into account which wheels are available on pypi for which python/subdir.

Here I would set everything below 3.11 to 1.21, because of this https://github.com/alkaline-ml/pmdarima/blob/v2.0.3/requirements.txt#LL3C4-L3C5